{"msg": "All items completed", "changed": true, "results": [{"_ansible_parsed": true, "cmd": "docker logs client-1", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:43 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-1/testdir-1/nfiles-98000-node-6", "Path: /cephfs-creates-1/testdir-1", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%", "", "1 tasks, 98000 files", "12/18/2017 17:17:42: Process 0(7999193d2949): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:05.248067", "_ansible_no_log": false, "item": "1", "stdout": "-- started at 12/18/2017 16:58:43 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-1/testdir-1/nfiles-98000-node-6\nPath: /cephfs-creates-1/testdir-1\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%\n\n1 tasks, 98000 files\n12/18/2017 17:17:42: Process 0(7999193d2949): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:05.219070", "delta": "0:00:00.028997", "stderr": "--------------------------------------------------------------------------\n[[46600,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 7999193d2949\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-1", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-2", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:44 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-2/testdir-2/nfiles-98000-node-6", "Path: /cephfs-creates-2/testdir-2", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%", "", "1 tasks, 98000 files", "12/18/2017 17:17:52: Process 0(bc2c2c64e5ed): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:05.499120", "_ansible_no_log": false, "item": "2", "stdout": "-- started at 12/18/2017 16:58:44 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-2/testdir-2/nfiles-98000-node-6\nPath: /cephfs-creates-2/testdir-2\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%\n\n1 tasks, 98000 files\n12/18/2017 17:17:52: Process 0(bc2c2c64e5ed): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:05.473175", "delta": "0:00:00.025945", "stderr": "--------------------------------------------------------------------------\n[[50687,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: bc2c2c64e5ed\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-2", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-3", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:45 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-3/testdir-3/nfiles-98000-node-6", "Path: /cephfs-creates-3/testdir-3", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%", "", "1 tasks, 98000 files", "12/18/2017 17:18:03: Process 0(7b460b749640): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:05.739941", "_ansible_no_log": false, "item": "3", "stdout": "-- started at 12/18/2017 16:58:45 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-3/testdir-3/nfiles-98000-node-6\nPath: /cephfs-creates-3/testdir-3\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%\n\n1 tasks, 98000 files\n12/18/2017 17:18:03: Process 0(7b460b749640): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:05.714530", "delta": "0:00:00.025411", "stderr": "--------------------------------------------------------------------------\n[[27250,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 7b460b749640\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-3", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-4", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:46 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-4/testdir-4/nfiles-98000-node-6", "Path: /cephfs-creates-4/testdir-4", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%", "", "1 tasks, 98000 files", "12/18/2017 17:18:13: Process 0(7febc9943339): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:05.992120", "_ansible_no_log": false, "item": "4", "stdout": "-- started at 12/18/2017 16:58:46 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-4/testdir-4/nfiles-98000-node-6\nPath: /cephfs-creates-4/testdir-4\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%\n\n1 tasks, 98000 files\n12/18/2017 17:18:13: Process 0(7febc9943339): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:05.969743", "delta": "0:00:00.022377", "stderr": "--------------------------------------------------------------------------\n[[29248,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 7febc9943339\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-4", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-5", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:47 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-5/testdir-5/nfiles-98000-node-6", "Path: /cephfs-creates-5/testdir-5", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%", "", "1 tasks, 98000 files", "12/18/2017 17:18:24: Process 0(66c94c99d128): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:06.230724", "_ansible_no_log": false, "item": "5", "stdout": "-- started at 12/18/2017 16:58:47 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-5/testdir-5/nfiles-98000-node-6\nPath: /cephfs-creates-5/testdir-5\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%\n\n1 tasks, 98000 files\n12/18/2017 17:18:24: Process 0(66c94c99d128): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:06.205480", "delta": "0:00:00.025244", "stderr": "--------------------------------------------------------------------------\n[[62443,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 66c94c99d128\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-5", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-6", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:48 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-6/testdir-6/nfiles-98000-node-6", "Path: /cephfs-creates-6/testdir-6", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%", "", "1 tasks, 98000 files", "12/18/2017 17:18:34: Process 0(1211a7861de3): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:06.499346", "_ansible_no_log": false, "item": "6", "stdout": "-- started at 12/18/2017 16:58:48 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-6/testdir-6/nfiles-98000-node-6\nPath: /cephfs-creates-6/testdir-6\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%\n\n1 tasks, 98000 files\n12/18/2017 17:18:34: Process 0(1211a7861de3): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:06.472535", "delta": "0:00:00.026811", "stderr": "--------------------------------------------------------------------------\n[[35167,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 1211a7861de3\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-6", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-7", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:49 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-7/testdir-7/nfiles-98000-node-6", "Path: /cephfs-creates-7/testdir-7", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%", "", "1 tasks, 98000 files", "12/18/2017 17:18:45: Process 0(4e1e7b12b945): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:06.758648", "_ansible_no_log": false, "item": "7", "stdout": "-- started at 12/18/2017 16:58:49 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-7/testdir-7/nfiles-98000-node-6\nPath: /cephfs-creates-7/testdir-7\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%\n\n1 tasks, 98000 files\n12/18/2017 17:18:45: Process 0(4e1e7b12b945): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:06.732154", "delta": "0:00:00.026494", "stderr": "--------------------------------------------------------------------------\n[[1603,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 4e1e7b12b945\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-7", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-8", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:49 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-8/testdir-8/nfiles-98000-node-6", "Path: /cephfs-creates-8/testdir-8", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%", "", "1 tasks, 98000 files", "12/18/2017 17:18:55: Process 0(44b8cef60a9f): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:07.011181", "_ansible_no_log": false, "item": "8", "stdout": "-- started at 12/18/2017 16:58:49 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-8/testdir-8/nfiles-98000-node-6\nPath: /cephfs-creates-8/testdir-8\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 105.0%\n\n1 tasks, 98000 files\n12/18/2017 17:18:55: Process 0(44b8cef60a9f): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:06.983749", "delta": "0:00:00.027432", "stderr": "--------------------------------------------------------------------------\n[[30305,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 44b8cef60a9f\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-8", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-9", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:50 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-9/testdir-9/nfiles-98000-node-6", "Path: /cephfs-creates-9/testdir-9", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 103.4%", "", "1 tasks, 98000 files", "12/18/2017 17:19:06: Process 0(51b439bfb8f3): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:07.252507", "_ansible_no_log": false, "item": "9", "stdout": "-- started at 12/18/2017 16:58:50 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-9/testdir-9/nfiles-98000-node-6\nPath: /cephfs-creates-9/testdir-9\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 103.4%\n\n1 tasks, 98000 files\n12/18/2017 17:19:06: Process 0(51b439bfb8f3): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:07.232140", "delta": "0:00:00.020367", "stderr": "--------------------------------------------------------------------------\n[[35927,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 51b439bfb8f3\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-9", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-10", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:51 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-10/testdir-10/nfiles-98000-node-6", "Path: /cephfs-creates-10/testdir-10", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 103.4%", "", "1 tasks, 98000 files", "12/18/2017 17:19:16: Process 0(dec4357b003e): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:07.500383", "_ansible_no_log": false, "item": "10", "stdout": "-- started at 12/18/2017 16:58:51 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-10/testdir-10/nfiles-98000-node-6\nPath: /cephfs-creates-10/testdir-10\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 103.4%\n\n1 tasks, 98000 files\n12/18/2017 17:19:16: Process 0(dec4357b003e): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:07.473056", "delta": "0:00:00.027327", "stderr": "--------------------------------------------------------------------------\n[[52307,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: dec4357b003e\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-10", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-11", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:52 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-11/testdir-11/nfiles-98000-node-6", "Path: /cephfs-creates-11/testdir-11", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 103.4%", "", "1 tasks, 98000 files", "12/18/2017 17:19:27: Process 0(807010e33200): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:07.738078", "_ansible_no_log": false, "item": "11", "stdout": "-- started at 12/18/2017 16:58:52 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-11/testdir-11/nfiles-98000-node-6\nPath: /cephfs-creates-11/testdir-11\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 103.4%\n\n1 tasks, 98000 files\n12/18/2017 17:19:27: Process 0(807010e33200): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:07.709671", "delta": "0:00:00.028407", "stderr": "--------------------------------------------------------------------------\n[[58517,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 807010e33200\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-11", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-12", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:53 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-12/testdir-12/nfiles-98000-node-6", "Path: /cephfs-creates-12/testdir-12", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 103.4%", "", "1 tasks, 98000 files", "12/18/2017 17:19:37: Process 0(0f1b1e2bfe8b): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:07.982021", "_ansible_no_log": false, "item": "12", "stdout": "-- started at 12/18/2017 16:58:53 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-12/testdir-12/nfiles-98000-node-6\nPath: /cephfs-creates-12/testdir-12\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 103.4%\n\n1 tasks, 98000 files\n12/18/2017 17:19:37: Process 0(0f1b1e2bfe8b): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:07.956030", "delta": "0:00:00.025991", "stderr": "--------------------------------------------------------------------------\n[[2085,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 0f1b1e2bfe8b\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-12", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-13", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:54 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-13/testdir-13/nfiles-98000-node-6", "Path: /cephfs-creates-13/testdir-13", "FS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 103.4%", "", "1 tasks, 98000 files", "12/18/2017 17:19:48: Process 0(9f53a11a5d54): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:08.224236", "_ansible_no_log": false, "item": "13", "stdout": "-- started at 12/18/2017 16:58:54 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-13/testdir-13/nfiles-98000-node-6\nPath: /cephfs-creates-13/testdir-13\nFS: 29.2 GiB   Used FS: 5.6%   Inodes: 0.0 Mi   Used Inodes: 103.4%\n\n1 tasks, 98000 files\n12/18/2017 17:19:48: Process 0(9f53a11a5d54): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:08.200374", "delta": "0:00:00.023862", "stderr": "--------------------------------------------------------------------------\n[[6571,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 9f53a11a5d54\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-13", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-14", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:55 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-14/testdir-14/nfiles-98000-node-6", "Path: /cephfs-creates-14/testdir-14", "FS: 29.2 GiB   Used FS: 5.8%   Inodes: 0.0 Mi   Used Inodes: 102.4%", "", "1 tasks, 98000 files", "12/18/2017 17:19:58: Process 0(71f0362da3a5): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:08.471830", "_ansible_no_log": false, "item": "14", "stdout": "-- started at 12/18/2017 16:58:55 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-14/testdir-14/nfiles-98000-node-6\nPath: /cephfs-creates-14/testdir-14\nFS: 29.2 GiB   Used FS: 5.8%   Inodes: 0.0 Mi   Used Inodes: 102.4%\n\n1 tasks, 98000 files\n12/18/2017 17:19:58: Process 0(71f0362da3a5): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:08.443746", "delta": "0:00:00.028084", "stderr": "--------------------------------------------------------------------------\n[[17938,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 71f0362da3a5\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-14", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-15", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:56 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-15/testdir-15/nfiles-98000-node-6", "Path: /cephfs-creates-15/testdir-15", "FS: 29.2 GiB   Used FS: 5.8%   Inodes: 0.0 Mi   Used Inodes: 102.4%", "", "1 tasks, 98000 files", "12/18/2017 17:20:09: Process 0(812231b326ac): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:08.717623", "_ansible_no_log": false, "item": "15", "stdout": "-- started at 12/18/2017 16:58:56 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-15/testdir-15/nfiles-98000-node-6\nPath: /cephfs-creates-15/testdir-15\nFS: 29.2 GiB   Used FS: 5.8%   Inodes: 0.0 Mi   Used Inodes: 102.4%\n\n1 tasks, 98000 files\n12/18/2017 17:20:09: Process 0(812231b326ac): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:08.693104", "delta": "0:00:00.024519", "stderr": "--------------------------------------------------------------------------\n[[8889,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 812231b326ac\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-15", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-16", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:57 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-16/testdir-16/nfiles-98000-node-6", "Path: /cephfs-creates-16/testdir-16", "FS: 29.2 GiB   Used FS: 5.8%   Inodes: 0.0 Mi   Used Inodes: 102.4%", "", "1 tasks, 98000 files", "12/18/2017 17:20:19: Process 0(d2ac551c7606): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:08.953548", "_ansible_no_log": false, "item": "16", "stdout": "-- started at 12/18/2017 16:58:57 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-16/testdir-16/nfiles-98000-node-6\nPath: /cephfs-creates-16/testdir-16\nFS: 29.2 GiB   Used FS: 5.8%   Inodes: 0.0 Mi   Used Inodes: 102.4%\n\n1 tasks, 98000 files\n12/18/2017 17:20:19: Process 0(d2ac551c7606): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:08.929153", "delta": "0:00:00.024395", "stderr": "--------------------------------------------------------------------------\n[[13548,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: d2ac551c7606\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-16", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-17", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:58 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-17/testdir-17/nfiles-98000-node-6", "Path: /cephfs-creates-17/testdir-17", "FS: 29.2 GiB   Used FS: 5.8%   Inodes: 0.0 Mi   Used Inodes: 102.4%", "", "1 tasks, 98000 files", "12/18/2017 17:20:30: Process 0(c613ec555e79): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:09.192800", "_ansible_no_log": false, "item": "17", "stdout": "-- started at 12/18/2017 16:58:58 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-17/testdir-17/nfiles-98000-node-6\nPath: /cephfs-creates-17/testdir-17\nFS: 29.2 GiB   Used FS: 5.8%   Inodes: 0.0 Mi   Used Inodes: 102.4%\n\n1 tasks, 98000 files\n12/18/2017 17:20:30: Process 0(c613ec555e79): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:09.165159", "delta": "0:00:00.027641", "stderr": "--------------------------------------------------------------------------\n[[7389,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: c613ec555e79\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-17", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-18", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:58:59 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-18/testdir-18/nfiles-98000-node-6", "Path: /cephfs-creates-18/testdir-18", "FS: 29.2 GiB   Used FS: 5.8%   Inodes: 0.0 Mi   Used Inodes: 102.4%", "", "1 tasks, 98000 files", "12/18/2017 17:20:40: Process 0(d4e53762cf82): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:09.435798", "_ansible_no_log": false, "item": "18", "stdout": "-- started at 12/18/2017 16:58:59 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-18/testdir-18/nfiles-98000-node-6\nPath: /cephfs-creates-18/testdir-18\nFS: 29.2 GiB   Used FS: 5.8%   Inodes: 0.0 Mi   Used Inodes: 102.4%\n\n1 tasks, 98000 files\n12/18/2017 17:20:40: Process 0(d4e53762cf82): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:09.410542", "delta": "0:00:00.025256", "stderr": "--------------------------------------------------------------------------\n[[49296,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: d4e53762cf82\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-18", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-19", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:59:00 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-19/testdir-19/nfiles-98000-node-6", "Path: /cephfs-creates-19/testdir-19", "FS: 29.2 GiB   Used FS: 6.0%   Inodes: 0.0 Mi   Used Inodes: 101.7%", "", "1 tasks, 98000 files", "12/18/2017 17:20:51: Process 0(7e57287ee9b2): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:09.675150", "_ansible_no_log": false, "item": "19", "stdout": "-- started at 12/18/2017 16:59:00 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-19/testdir-19/nfiles-98000-node-6\nPath: /cephfs-creates-19/testdir-19\nFS: 29.2 GiB   Used FS: 6.0%   Inodes: 0.0 Mi   Used Inodes: 101.7%\n\n1 tasks, 98000 files\n12/18/2017 17:20:51: Process 0(7e57287ee9b2): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:09.646615", "delta": "0:00:00.028535", "stderr": "--------------------------------------------------------------------------\n[[48961,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: 7e57287ee9b2\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-19", "removes": null, "warn": true, "chdir": null}}}, {"_ansible_parsed": true, "cmd": "docker logs client-20", "warnings": [], "_ansible_item_result": true, "changed": true, "stdout_lines": ["-- started at 12/18/2017 16:59:01 --", "", "mdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)", "Command line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-20/testdir-20/nfiles-98000-node-6", "Path: /cephfs-creates-20/testdir-20", "FS: 29.2 GiB   Used FS: 6.0%   Inodes: 0.0 Mi   Used Inodes: 101.7%", "", "1 tasks, 98000 files", "12/18/2017 17:21:01: Process 0(c60271dfa07e): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort"], "end": "2017-12-18 11:21:09.904154", "_ansible_no_log": false, "item": "20", "stdout": "-- started at 12/18/2017 16:59:01 --\n\nmdtest-1.9.3 was launched with 1 total task(s) on 1 node(s)\nCommand line used: /mdtest/mdtest -F -C -n 98000 -d /cephfs-creates-20/testdir-20/nfiles-98000-node-6\nPath: /cephfs-creates-20/testdir-20\nFS: 29.2 GiB   Used FS: 6.0%   Inodes: 0.0 Mi   Used Inodes: 101.7%\n\n1 tasks, 98000 files\n12/18/2017 17:21:01: Process 0(c60271dfa07e): FAILED in create_remove_items_helper, unable to create file: Software caused connection abort", "start": "2017-12-18 11:21:09.881575", "delta": "0:00:00.022579", "stderr": "--------------------------------------------------------------------------\n[[53274,1],0]: A high-performance Open MPI point-to-point messaging module\nwas unable to find any relevant network interfaces:\n\nModule: OpenFabrics (openib)\n  Host: c60271dfa07e\n\nAnother transport will be used instead, although this may result in\nlower performance.\n--------------------------------------------------------------------------\n--------------------------------------------------------------------------\nMPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD \nwith errorcode 1.\n\nNOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.\nYou may or may not see output from other processes, depending on\nexactly when Open MPI kills them.\n--------------------------------------------------------------------------", "rc": 0, "invocation": {"module_name": "command", "module_args": {"creates": null, "executable": null, "_uses_shell": true, "_raw_params": "docker logs client-20", "removes": null, "warn": true, "chdir": null}}}]}